components/runtime/vala/Makefile

```diff
@@ -19,21 +19,28 @@
 # CDDL HEADER END
 #
 # Copyright (c) 2013 David Hoeppner. All rights reserved.
+# Copyright (c) 2017 Andreas Wacknitz
 #
-include ../../make-rules/shared-macros.mk
+include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		vala
 COMPONENT_VERSION=	0.20.1
+COMPONENT_REVISION=	1
+COMPONENT_LICENSE=	LGPL2.1
+COMPONENT_LICENSE_FILE=	vala.license
+COMPONENT_FMRI=		runtime/vala
```

Comment:
Ok, I can revert this. I thought it was a good idea because I moved vala to the runtime folder. Maybe I should move it to developer?